tracardi/process_engine/action/v1/internal/get_prev_event/plugin.py

```diff
@@ -67,7 +67,9 @@ def register() -> Plugin:
                                 id="event_type",
                                 description="Please provide event type that you want to load into payload. If you want "
                                             "current but previous event, insert @current.",
-                                component=FormComponent(type="eventType", props={"label": "Event type"})
+                                component=FormComponent(type="eventType", props={"label": "Event type", "props": {
+                                    "defaultValueSet": [{"id": "@current", "name": "@current"}]
```

Comment:
I do not think it solves the problem. Did you add some events to the database. I guess it will override your settings.

Reply:
It does solve the problem. GUI part has to be merged as well in order for it to work. That's default AutoComplete feature, which was not available due to not passing props down JSON form component. Now it does so.